src/include/distributed/commands/multi_copy.h

```diff
@@ -133,6 +133,9 @@ typedef struct CitusCopyDestReceiver
 	/* if true, should copy to local placements in the current session */
 	bool shouldUseLocalCopy;
 
+	/* if true, the operations in the receiver can be published.*/
```

Comment:
I think we could improve this comment, the term "Publishable" doesn't seem to be documented anywhere, and this place could be a good one

Reply:
OK. I have added the comments.